### PR TITLE
updating code example

### PR DESCRIPTION
**Pull request by apigirl**
### What does this PR do?
updates Java dd trace and OpenTracing example - [preview link](https://docs-staging.datadoghq.com/kaylyn/opentracing/tracing/advanced/opentracing/?tab=java#pagetitle)

### CODE REVIEW

File: content/en/tracing/advanced/opentracing.md [hunk 1]
```diff
@@ -125,29 +125,31 @@ If you’re not using `dd-java-agent.jar`, you must register a configured tracer
...
-import datadog.trace.api.sampling.AllSampler;
-import datadog.trace.common.writer.DDAgentWriter;
-
 import io.opentracing.Tracer;
 import io.opentracing.util.GlobalTracer;
+import datadog.trace.common.sampling.AllSampler;
+import datadog.trace.common.writer.DDAgentWriter;
+
+//For the API Example
+        import datadog.trace.common.writer.Writer;
```
Comment (l0k0ms) [inline content/en/tracing/advanced/opentracing.md]:
```suggestion
import datadog.trace.common.writer.Writer;
```

File: content/en/tracing/advanced/opentracing.md [hunk 2]
```diff
@@ -125,29 +125,31 @@ If you’re not using `dd-java-agent.jar`, you must register a configured tracer
...
+        import datadog.trace.common.sampling.Sampler;
```
Comment (l0k0ms) [inline content/en/tracing/advanced/opentracing.md]:
```suggestion
import datadog.trace.common.sampling.Sampler;
```

File: content/en/tracing/advanced/opentracing.md [hunk 3]
```diff
@@ -125,29 +125,31 @@ If you’re not using `dd-java-agent.jar`, you must register a configured tracer
...
 
 public class Application {
 
     public static void main(String[] args) {
 
-        // Initialize the tracer from environment variables or system properties
-        Tracer tracer = new DDTracer();
+//         Initialize the tracer from environment variables or system properties
```
Comment (labbati) [inline content/en/tracing/advanced/opentracing.md]: You might want to bring this back to indented comment, as it was?

File: content/en/tracing/advanced/opentracing.md [hunk 4]
```diff
@@ -125,29 +125,31 @@ If you’re not using `dd-java-agent.jar`, you must register a configured tracer
...
+        DDTracer tracer = new DDTracer();
         GlobalTracer.register(tracer);
         // register the same tracer with the Datadog API
         datadog.trace.api.GlobalTracer.registerIfAbsent(tracer);
 
-        // OR from the API
+//      OR from the API
```
Comment (labbati) [inline content/en/tracing/advanced/opentracing.md]: Same as above

File: content/en/tracing/advanced/opentracing.md [hunk 5]
```diff
@@ -125,29 +125,31 @@ If you’re not using `dd-java-agent.jar`, you must register a configured tracer
...
+import datadog.trace.common.writer.Writer;
+import datadog.trace.common.sampling.Sampler;
 
 public class Application {
 
     public static void main(String[] args) {
 
-        // Initialize the tracer from environment variables or system properties
-        Tracer tracer = new DDTracer();
+//         Initialize the tracer from environment variables or system properties
```
Comment (apigirl) [inline content/en/tracing/advanced/opentracing.md]:
```suggestion
          // Initialize the tracer from environment variables or system properties
```

File: content/en/tracing/advanced/opentracing.md [hunk 6]
```diff
@@ -125,29 +125,31 @@ If you’re not using `dd-java-agent.jar`, you must register a configured tracer
...
+        DDTracer tracer = new DDTracer();
         GlobalTracer.register(tracer);
         // register the same tracer with the Datadog API
         datadog.trace.api.GlobalTracer.registerIfAbsent(tracer);
 
-        // OR from the API
+//      OR from the API
```
Comment (apigirl) [inline content/en/tracing/advanced/opentracing.md]:
```suggestion
        // OR from the API
```

File: content/en/tracing/advanced/opentracing.md [hunk 7]
```diff
@@ -125,29 +125,31 @@ If you’re not using `dd-java-agent.jar`, you must register a configured tracer
...
+import datadog.trace.common.writer.Writer;
+import datadog.trace.common.sampling.Sampler;
 
 public class Application {
 
     public static void main(String[] args) {
 
-        // Initialize the tracer from environment variables or system properties
-        Tracer tracer = new DDTracer();
+          // Initialize the tracer from environment variables or system properties
```
Comment (l0k0ms) [inline content/en/tracing/advanced/opentracing.md]:
```suggestion
        // Initialize the tracer from environment variables or system properties
```

**Pull request merged**